### ISSUE TITLE
Add a leading character

### ISSUE THREAD

**Issue by svenstaro** [FreeMasen/progress_string#2]
wget's progress bar looks like this:

```
l_mkl_2020.2.254.tgz.1                              14%[===============>                                                                                                 ]  74.55M   750KB/s    eta 13m 22s^C
```

You can see it has this leading `>` character. I think it'd be cool if your library supported this kind of leading character.

**Comment by svenstaro** [comment]
@FreeMasen any chance I can trick you into implementing this tiny feature? :)

**Comment by FreeMasen** [comment]
It seems like a good feature. I will see if I can't get it together.

**Comment by FreeMasen** [comment]
Thanks for the request, sorry it took so long! This is now merged and published as v0.2.1
